packages/zosjobs/src/SubmitJobs.ts

```diff
@@ -106,16 +106,23 @@ export class SubmitJobs {
      * @returns {Promise<IJob>} - Promise that resolves to an IJob document with details about the submitted job
      * @memberof SubmitJobs
      */
-    public static submitJcl(session: AbstractSession, jcl: string, internalReaderRecfm?: string, internalReaderLrecl?: string) {
+    public static submitJcl(session: AbstractSession, jcl: string, internalReaderRecfm?: string,
```

Comment:
Same as previous commnet...
Should we move everything after `jcl` (or include `jcl` too) into a typed object (with its own interface)?
This can be done in V3, and we can follow the example of `submitJobCommon` in this file 😋

Reply:
...however, on this API, we already call submitJclCommon. Frankly, I think it would be nice to just get rid of some of these redundant APIs.